src/Components/RecsListTable/RecsListTable.spec.ct.js

```diff
@@ -2,26 +2,108 @@ import React from 'react';
 import { mount } from '@cypress/react';
 import { MemoryRouter } from 'react-router-dom';
 import { Provider } from 'react-redux';
+import _ from 'lodash';
 
 import { RecsListTable } from './RecsListTable';
 import getStore from '../../Store';
-import data from '../../../cypress/fixtures/api/insights-results-aggregator/v2/rule.json';
+import ruleResponse from '../../../cypress/fixtures/api/insights-results-aggregator/v2/rule.json';
 import { Intl } from '../../Utilities/intlHelper';
 import '@patternfly/patternfly/patternfly.scss';
 import {
   TOOLBAR,
   TOOLBAR_FILTER,
   CHIP,
   CHIP_GROUP,
+  PAGINATION,
 } from '../../../cypress/utils/components';
 import { urlParamConvert } from '../../../cypress/utils/filters';
+import {
+  DEFAULT_ROW_COUNT,
+  PAGINATION_VALUES,
+} from '../../../cypress/utils/defaults';
+import {
+  checkPaginationTotal,
+  checkPaginationValues,
+  changePagination,
+} from '../../../cypress/utils/pagination';
+import { TOTAL_RISK, CATEGORIES } from '../../../cypress/utils/globals';
 // TODO make more use of ../../../cypress/utils/components
 
 // selectors
 const ROOT = 'div[id=recs-list-table]';
 const ROW = 'tbody[role=rowgroup]'; // FIXME use ROW from components
 const FILTERS_DROPDOWN = 'ul[class=pf-c-dropdown__menu]';
 const FILTER_TOGGLE = 'span[class=pf-c-select__toggle-arrow]';
+const DEFAULT_FILTERS = {
```

Comment:
Maybe worth to refer to https://github.com/RedHatInsights/ocp-advisor-frontend/blob/master/src/Services/Filters.js#L13

Reply:
Makes sense. However, as some of them are not useful for filtering the data, it might require some thinking. I will create a `TODO` and address it separately. Thanks for pointing it out.